### PR TITLE
TINY-8585: Removed "Added in TinyMCE" version comments, updated "setEnabled" and "isEnabled" API comments

### PR DESCRIPTION
**Pull request by MitchCTiny**
Related Ticket: TINY-8585

Description of Changes:
* Removed instances of "Added in TinyMCE 5.x"
* Updated renamed enabled API documentation
* Only comment blocks have been altered

Pre-checks:
* [x] ~~Changelog entry added~~
* [x] ~~Tests have been added (if applicable)~~
* [x] Branch prefixed with `feature/` for new features (if applicable)

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

**Comment by lnewson** [comment]
Looks like one of the Added in comments was missed here: https://github.com/tinymce/tinymce/blob/236891ffcdf2fd1b698d020e67c31838fb110019/modules/tinymce/src/core/main/ts/api/util/ImageUploader.ts#L32

File: modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js [hunk 1]
```diff
@@ -29,44 +27,28 @@
...
- *
- * @method tinymce.editor.ui.enable
- */
-
-/**
- * Disables the editor user interface. This method affects all user interface
- * elements, including: menu bar, toolbar, notifications, and dialogs.
- * <br>
- * <em>Added in TinyMCE 5.6</em>
+ * Sets the state of editor user interface as `enabled` (`true`) or not (`false`).
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js]:
```suggestion
 * Sets the state of editor user interface as `enabled` (`true`) or disabled (`false`).
```

File: modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js [hunk 2]
```diff
@@ -29,44 +27,28 @@
...
+ * This method affects all user interface elements, including: menu bar,
+ * toolbar, notifications, and dialogs. Can not be set to 'true' when in readonly mode.
  *
- * @method tinymce.editor.ui.disable
+ * @method tinymce.editor.ui.setEnabled
+ * @parameters {Boolean} true/false the state in which to set the user interface
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js]:
```suggestion
 * @param {Boolean} state The state in which to set the user interface.
```

File: modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js [hunk 3]
```diff
@@ -29,44 +27,28 @@
...
  */
 
 /**
- * Determines if the editor user interface is `disabled` (`true`) or not (`false`).
- * <br>
- * <em>Added in TinyMCE 5.6</em>
+ * Determines if the editor user interface is `enabled` (`true`) or not (`false`).
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js:44]:
```suggestion
 * Determines if the editor user interface is `enabled` (`true`) or disabled (`false`).
```

File: modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js [hunk 4]
```diff
@@ -29,44 +27,28 @@
...
  *
- * @method tinymce.editor.ui.isDisabled
- * @return {Boolean} true/false if the editor user interface is `disabled` (`true`) or not (`false`).
+ * @method tinymce.editor.ui.isEnabled
+ * @return {Boolean} true/false if the editor user interface is `enabled` (`true`) or not (`false`).
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.editor.ui.Ui.js:47]:
```suggestion
 * @return {Boolean} true/false if the editor user interface is enabled (`true`) or disabled (`false`).
```

**Review by lnewson — APPROVED** [review]

**Review by TheSpyder — APPROVED** [review]

**Pull request merged**